src/foam/nanos/cron/GenericIntervalSchedule.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.cron',
+  name: 'GenericIntervalSchedule',
```

Comment:
Please rename
```suggestion
  name: 'GenericIntervalSchedule',
```
options:
IntervalSchedule
DateFrequencySchedule
SimpleIntervalSchedule
anything but 'Generic'.

Reply:
Renamed to SimpleIntervalSchedule since IntervalSchedule already taken by the hms intervalSchedule